lib/rules/space-after-keywords.js

```diff
@@ -23,6 +23,10 @@ module.exports = function(context) {
      * @returns {void}
      */
     function checkTokens(node, left, right) {
+        if (right.type !== "Punctuator") {
```

Comment:
Not sure how this would increase the warnings created since it actually only filters any existing warnings. Any duplicate warnings should already be in the code and I'd argue fixing that is not in the scope of this issue/PR.